resin/twofactor_auth.py

```diff
@@ -0,0 +1,99 @@
+import urlparse
+
+import pyotp
+
+from .base_request import BaseRequest
+from .token import Token
+from .settings import Settings
+from . import exceptions
+
+
+class TwoFactor_Auth(object):
+    """
+    This class implements basic 2FA functionalities for Resin Python SDK.
+
+    """
+
+    def __init__(self):
+        self.base_request = BaseRequest()
+        self.settings = Settings()
+        self.token = Token()
+
+    def is_enabled(self):
+        """
+        Check if two-factor authentication is enabled.
+
+        Returns:
+            bool: True if enabled. Otherwise False.
+
+        """
+
+        try:
+            self.token.get_property('twoFactorRequired')
+            return True
```

Comment:
I assume ``get_property`` returns a string, right? To be on the safe side, I'd suggest you try to parse the value whether it's '1', 'true' or 'on' and not default to returning ``True`` just because the property exists -- it can accidentally be set to a falsy value